Server/src/main/resources/oxauth_de.properties

```diff
@@ -195,3 +195,15 @@ password.validation.invalid = The password provide is not strong enough
 
 # CIBA
 ciba.bindingMessage=Binding Message
+
```

Comment:
What is the sense of pasting english into oxauth_de file? Resource bundle automatically falls back to oxauth.properties if key is not found. Same for other localionzation files.